src/actors/cficaches.rs

```diff
@@ -1,12 +1,15 @@
 use std::fs::File;
 use std::io::{self, BufWriter};
 use std::path::Path;
+use std::pin::Pin;
 use std::sync::Arc;
 use std::time::Duration;
 
-use futures::{compat::Future01CompatExt, FutureExt, TryFutureExt};
-use futures01::future::{Either, Future, IntoFuture};
-use sentry::configure_scope;
+use futures::compat::Future01CompatExt;
+use futures::future::{self, Either};
+use futures::{Future, FutureExt, TryFutureExt};
+// use futures01::future::{Either, Future, IntoFuture};
```

Comment:
```suggestion
```

Reply:
lol, i even specifically looked for things like this before committing... ah well.